aspnetcore/tutorials/index.md

```diff
@@ -16,6 +16,7 @@ ms.prod: asp.net-core
 The following step-by-step guides for developing ASP.NET Core applications are available:
 
 ## Building web applications
+* [Introduction to ASP.NET Core MVC on Mac or Linux ](xref:tutorials/first-mvc-app-xplat/start-mvc)
```

Comment:
I Intro to is now the preferred title, consider changing the VS one's title to match.  They should be parallel so it's clear the difference is the IDE/CLI use, not the content.